src/modules/stock.js

```diff
@@ -87,6 +111,21 @@ const stock = (state = initState, action) => {
       milk: Number(state.milk) + Number(quantity),
     };
   }
+  if (action.type === ADD_COKE) {
+    const { quantity } = action.payload;
+    return {
+      ...state,
+      coke: Number(state.coke) + Number(quantity),
```

Comment:
```suggestion
      coke: state.coke + quantity,
```
이미 둘 다 number 타입인 것 같습니다!!